fw-child/resources/js/child-functions.js

```diff
@@ -708,6 +709,25 @@ const $ = jQuery;
 
     };
 
+    //
+    // Dynamic logo block display scroll behavior
+    //
+    //
+
+    if ($('#floating-header').length && $('#hero').length) {
```

Comment:
```suggestion
    if ( $( '#floating-header' ).length && $( '#hero' ).length ) {
```
Please apply the same spacing style as suggested here everywhere